README.md

```diff
@@ -42,23 +42,11 @@ By Composer
 
 The easiest way to install PredictionIO PHP client is to use [Composer](http://getcomposer.org/).
 
-1. Add `predictionio/predictionio` as a dependency in your project's ``composer.json`` file:
+1. `Predictionio` is available on [Packagist](https://packagist.org/packages/league/csv) and can be installed using [Composer](https://getcomposer.org/):
```

Comment:
Do you mean https://packagist.org/ instead for Packagist's link?

The correct letter case for the module name should be `predictionio`. If indicating the project name, please use Apache PredictionIO.